File: beginner/pagina07/2165/2165.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException {
        Scanner input = new Scanner(System.in);

        String text = input.nextLine();

        if(text.length() <= 140){
            System.out.println("TWEET");
        } else {
            System.out.println("MUTE");
        }

        input.close();
    }
}

File: beginner/__2000__/2165/2165.cpp
#include <iostream>

using namespace std;

int main(){
    string text;
    cin >> text;

    if(text.size(0) <= 140){
        cout << "TWEET" << endl;
    } else {
        cout << "MUTE" << endl;
    }
    
    return 0;
}